src/comp/comp.py

```diff
@@ -32,7 +32,7 @@ def __repr__(self):
 a = []
 for x in humans:
     if x.name[0] == "D":
-        a.append(x)
+        a.append(x.name)
 print(a)
 
```

Comment:
These all look great Jess. Check out this snippet below:
``` python
for item in list:
    if conditional:
        expression
# This is equivalent to:
[ expression for item in list if conditional ]
```
When you have time go ahead and add the comprehension syntax using your existing code.
[List Comprehensions in Python](https://www.pythonforbeginners.com/basics/list-comprehensions-in-python)